scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -106,9 +111,18 @@ void unlock() {
 
     private final StartAtUpdater startAtUpdater = new StartAtUpdater();
 
-    LiveJobs(SchedulerDBManager dbManager, SchedulerStateUpdate listener) {
+    private final SynchronizationInternal synchronizationInternal;
+
+    private static final String SIGNAL_ORIGINATOR = "SCHEDULER";
+
+    private static final String SIGNAL_TASK = "0000000t0";
```

Comment:
why so many zeros ? if it's supposed to represent a fake task id, would "0t0" be sufficient ?